src/pages/scientificServices/landingPage/ScientificServicesDescription.tsx

```diff
@@ -15,7 +15,14 @@ export const ScientificServicesDescription = () => {
       </div>
       <div style={{ fontWeight: 'bold', marginTop: '2rem' }}>Our current offerings:</div>
       <div>
-        <span style={{ fontStyle: 'italic' }}>All of Us</span> + AnVIL Imputation Service
+        <a
```

Comment:
This is an unrelated drive-by change to link to the marketing page from the landing page